cmd/keytransparency-sequencer/main.go

```diff
@@ -63,7 +61,8 @@ var (
 	etcdServers = flag.String("etcd_servers", "", "A comma-separated list of etcd servers; no etcd registration if empty")
 	lockDir     = flag.String("lock_file_path", "/keytransparency/master", "etcd lock file directory path")
 
-	serverDBPath = flag.String("db", "db", "Database connection string")
+	dbPath   = flag.String("db", "", "Database connection string")
+	dbEngine = flag.String("db_engine", "mysql", "Storage implementation. One of ['mysql', 'spanner']")
```

Comment:
There's a bit of a layer violation here that the supported types are  hardcoded into this docstring but the impl switching is in another file. I don't have suggestions to improve this without making things more complicated though so happy to leave this if you are.

Reply:
Added a `StorageEngines()` function for this